test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey-cql-stress.yaml

```diff
@@ -0,0 +1,34 @@
+test_duration: 750
+prepare_write_cmd: "cql-stress-cassandra-stress write cl=ALL n=50050075  -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..50050075 -log interval=15"
+
+#stress_cmd: ["cql-stress-cassandra-stress write cl=QUORUM duration=11h  -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate 'threads=250 throttle=45000/s'  -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=400200300..600200300 -log interval=15"]
```

Comment:
why this commented out?

Reply:
still in progress - I was testing if this does not cause us troubles, but I found root cause https://github.com/scylladb/cql-stress/issues/138